### PR TITLE
[MIG] website_sale_product_attachment: Migration to 14.0

### PR DESCRIPTION
**Pull request by Zar21**
Standard migration Odoo 14.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please include a93cb60784da99488a53b9009e5fcc1765c54716 that I have just discovered and fixed for v13.

**Comment by Zar21** [comment]
Hi @pedrobaeza,
I added your fix in 13.0 version.
Thanks.

**Comment by pedrobaeza** [comment]
Great, thanks.

**Comment by francesco-ooops** [comment]
@Zar21 can you rebase for testing? 

thanks

File: website_sale_product_attachment/templates/product_template.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                     >
                         <ul class="list-group list-group-flush">
                             <t
-                                t-foreach="product.website_attachment_ids"
+                                t-foreach="product.website_attachment_ids.sudo()"
```
Comment (chienandalu) [inline website_sale_product_attachment/templates/product_template.xml:43]: Have security rules changed from v13?

**Review by chienandalu — COMMENTED** [review]
Rebase with the most recent 14.0 branch to enjoy new CIs and make functional testing possible

**Comment by fuentes73** [comment]
@miguel-S73 could your rebase?

File: website_sale_product_attachment/templates/product_template.xml (hunk 1 above)
Comment (pedrobaeza) [reply website_sale_product_attachment/templates/product_template.xml:43]: Any answer to this question?

**Pull request closed without merging**